src/NuGet.Core/NuGet.Configuration/PackageSourceMapping/PackageSourceMapping.cs

```diff
@@ -18,7 +18,7 @@ public class PackageSourceMapping
         /// <summary>
         /// Source name to package patterns list.
         /// </summary>
-        internal IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
+        public IReadOnlyDictionary<string, IReadOnlyList<string>> Patterns { get; }
```

Comment:
This PR is not in draft mode, but this change (and others in `src/NuGet.Core`) are from this PR: https://github.com/NuGet/NuGet.Client/pull/4709

Please focus one completing dependent PRs before marking new PRs that need the same change as "ready for review", to minimize workload on the rest of the team.